### ISSUE TITLE
Trailing semicolon not detected with Windows newlines

### ISSUE THREAD

**Issue by provegard** [clutchski/coffeelint#57]
The following file with a Windows newline (CR+LF) after the semicolon passes trailing semicolon detection:

```
x = 5;

```

```
  V test.coffee

V Ok! » 0 errors and 0 warnings in 1 file
```

If I change to a Unix newline, I get:

```
     ? #1: Line contains a trailing semicolon.

? Lint! » 1 error and 0 warnings in 1 file
```
